attendance/API_HR/c-hr.php

```diff
@@ -2376,7 +2409,15 @@ public static function addNewEmployee($PARAMS) { //api call
                         ( '$f_name', '$f_jobtitle', '$f_dateofjoining', $userID, '$f_dob', '$f_gender', '$f_workemail', $f_training_month ) ";
                     self::DBrunQuery($q1);
                     $r_error = 0;
-                    $r_message = "Employee added Successfully !!";
+                    
+                    //$r_message = "Employee added Successfully !!";
+
+                    // Added on 15-03-18 to send Welcome mail to new user
+                    if ($userID == true) {
+                        $pr_message = "Employee added Successfully !!";
+                        $tr_message = self::sendWelcomeMail($mailmsg); // call welcome mail
+                        $r_message = $pr_message." ".$tr_message;  
```

Comment:
why did u concat this tr_message to pr_message?

Reply:
sir, to throw a pop up message that will show "employee added successfully. Email is sent"